### DISCUSSION TITLE
Potential Typo?

### DISCUSSION THREAD

**Discussion by AkshayPatil1994** [NCAR/FastEddy-model#54] [Installation/Build and Configuration]
https://github.com/NCAR/FastEddy-model/blob/4ee44512b826ce854a5ba9365f77b818e09cc990/SRC/FEMAIN/Makefile#L141

Original version
```
../TIME_INTEGBRATION/CUDA/cuda_timeIntDevice.o: ../CUDA/cuda_timeIntDevice.cu \
        ../TIME_INTEGRATION.CUDA/cuda_RKschemes.cu
```

Fixed version
```
../TIME_INTEGRATION/CUDA/cuda_timeIntDevice.o: ../TIME_INTEGRATION/CUDA/cuda_timeIntDevice.cu \
        ../TIME_INTEGRATION/CUDA/cuda_RKschemes.cu
```

I tried to compile FastEddy on my workstation with a CUDA compiler and noticed that it could not find `#include <mpi.h>` potentially due to a typo.

Regards,
Akshay P.

**Comment by jprestop** [comment]
@AkshayPatil1994 Thank you for your feedback. We will review it as time permits.

**Comment by jsauer-NCAR** [comment]
Thank you for the feedback in pointing out this issue. This has been corrected in the FastEddy v4.0 release.
